### ISSUE TITLE
Implement Class Methods.

### ISSUE THREAD

**Issue by mpekim** [mpekim/hello-world#5]
In order for the Dog class to be functional, we need to implement methods that the class can use.

**Comment by mpekim** [comment]
**8/4:** - Sabbatical Day. No progress has been made on the issue.

**Comment by mpekim** [comment]
**8/13:** Day off, going out. Also, prep for the Fall 2022 semester is more important.

**Comment by mpekim** [comment]
**8/18:** Cat object added to the code.

**Comment by mpekim** [comment]
**1/14:** Implemented Element Object code to print Major, Minor and Warning errors to index.html file.

**Comment by mpekim** [comment]
**5/21:** Sick day.

**Comment by mpekim** [comment]
**12/15:** At this time, error messages and Final Output for JS code is presented on *index.html* file.

**Comment by mpekim** [comment]
**12/25:** Merry Christmas!

**Comment by mpekim** [comment]
**3/11:** Still waiting on response from IT company.

**Comment by mpekim** [comment]
**3/15:** Complete work for Web Design project (RAVAGE).

**Comment by mpekim** [comment]
**7/6:** as of this time, the following methods work: constructor, destructor, accessor and mutator, presentation.

**Comment by mpekim** [comment]
**7/24:** Debating to close issue.

**Comment by mpekim** [comment]
**9/23:** Still trying to find final methods to implement.

**Comment by mpekim** [comment]
**10/4:** Finishing up CSS/HTML templates for remote work.

**Comment by mpekim** [comment]
**10/14:** Getting next push ready.

**Comment by mpekim** [comment]
**10/19:** About to make push to close issue.

**Comment by mpekim** [comment]
**10/24:** Trying to make language selection menu work.

**Comment by mpekim** [comment]
**11/1:** Debating to close issue.

**Comment by mpekim** [comment]
**11/10:** Learning Typescript to help with Angular work.

**Comment by mpekim** [comment]
**11/17:** Typescript and Angular certificates are coming soon.

**Comment by mpekim** [comment]
**4/18:** Preparing for Easter updates to remote work site.

**Comment by mpekim** [comment]
**4/24:** Debating to close issue.

**Comment by mpekim** [comment]
**5/5:** Issue can be closed soon.

**Comment by mpekim** [comment]
**5/15:** Be sure to cash out on remote work.

**Comment by mpekim** [comment]
**5/18:** Check and make sure all methods are implemented/tested before closing issue.

**Comment by mpekim** [comment]
**5/27:** Happy Memorial Day!

**Comment by mpekim** [comment]
**5/26:** Happy Memorial Day!

**Comment by mpekim** [comment]
**6/8:** Illness finally over; returning to remote work.

**Comment by mpekim** [comment]
**6/16:** Catching up on remote work.

**Comment by mpekim** [comment]
**6/24:** Trying to find any last methods to add before closing issue.

**Comment by mpekim** [comment]
**7/1:** Putting remote work on hold due to vacation.

**Comment by mpekim** [comment]
**7/8:** Almost finished with next push. (Remote work)

**Comment by mpekim** [comment]
**7/14:** Issue should be closed upon next push.

**Comment by mpekim** [comment]
**7/22:** Finalizing changes needed to close issue.

**Comment by mpekim** [comment]
**8/1:** Remote work is overdue; please catch up as soon as possible.

**Comment by mpekim** [comment]
**8/11:** Found solution to random password generator; empty input uses the null terminator as a single character “string”.

**Comment by mpekim** [comment]
**10/8:** Further update on random password generator. Conditional was based on assignment, not equality comparison.

**Comment by mpekim** [comment]
**10/15:** Issue can be closed soon.

**Comment by mpekim** [comment]
**10/27:** Closing issue after next push.

**Comment by mpekim** [comment]
**11/30:** Performing tests before closing issue.

**Comment by mpekim** [comment]
**12/17:** All tests work as expected. Issue can be closed soon.

**Comment by mpekim** [comment]
**12/25:** Merry Christmas!
